x-pack/legacy/plugins/rollup/public/crud_app/services/documentation_links.js

```diff
@@ -9,7 +9,7 @@ let xPackBase = '';
 
 export function setEsBaseAndXPackBase(elasticWebsiteUrl, docLinksVersion) {
   esBase = `${elasticWebsiteUrl}guide/en/elasticsearch/reference/${docLinksVersion}`;
-  xPackBase = `${elasticWebsiteUrl}guide/en/x-pack/${docLinksVersion}`;
+  xPackBase = `${elasticWebsiteUrl}guide/en/elasticsearch/reference/${docLinksVersion}`;
```

Comment:
Rather than changing the value of `xPackBase`, can you use `esBase` on L11 and delete the `xpackBase` variable if it's no longer being used?

Reply:
Sure can. I also saw this and was curious if that was the correct/better approach here. Nonetheless, I just pushed an updated branch with the requested changes. Happy Monday.